src/components/RightSidebar/RightSidebar.vue

```diff
@@ -69,9 +69,9 @@
 				:name="conversation.displayName" />
 			<div v-if="!getUserId" id="app-settings">
 				<div id="app-settings-header">
-					<button class="settings-button" @click="showSettings">
+					<Button type="tertiary" class="settings-button" @click="showSettings">
```

Comment:
class to be removed, at least for the styling of the element

<img width="357" alt="Screenshot 2022-03-07 at 18 09 54" src="https://user-images.githubusercontent.com/26852655/157083367-1e2231d0-48c8-456e-bcf5-2693572a0b72.png">

Reply:
Should I remove the class from new Buttons? :)